aea/cli/config.py

```diff
@@ -342,3 +343,23 @@ def convert_value_str_to_type(value: str, type_str: str) -> JSON_TYPES:
             return type_(value)
         except (ValueError, json.decoder.JSONDecodeError):  # pragma: no cover
             raise ValueError("Cannot convert {} to type {}".format(value, type_))
+
+    def get_overridables(self) -> Tuple[Dict, Dict[ComponentId, Dict]]:
+        """Get config overridables."""
+        agent_overridable = self.agent_config.get_overridable()
+
+        components_overridables: Dict[ComponentId, Dict] = {}
+        for component_id in self.agent_config.all_components_id:
+            obj = {}
+            component_config = self.load_component_configuration(
+                component_id, skip_consistency_check=True
+            )
+            obj.update(component_config.get_overridable())
+            obj.update(
+                deepcopy(
+                    self.agent_config.component_configurations.get(component_id, {})
+                )
+            )
+            if obj:
```

Comment:
How can it not be true?

Reply:
protocol has no overridables so set of overridables can be empty, checked  during tests